src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -395,4 +372,64 @@ describe("calcite-input-time-picker", () => {
     expect(await inputTimePicker.getProperty("value")).toBe("11:00:00");
     expect(await input.getProperty("value")).toBe("11:00 AM");
   });
+
+  describe("toggling time picker", () => {
+    let page: E2EPage;
+    let inputTimePicker: E2EElement;
+
+    beforeEach(async () => {
+      page = await newE2EPage();
+      await page.setContent(html` <calcite-input-time-picker></calcite-input-time-picker>`);
+      await skipAnimations(page);
+      await page.waitForChanges();
+      inputTimePicker = await page.find("calcite-input-time-picker");
+    });
+
+    it("does not open the time picker on input keyboard focus", async () => {
+      const popover = await page.find("calcite-input-time-picker >>> calcite-popover");
+
+      await page.keyboard.press("Tab");
+      await page.waitForChanges();
+
+      expect(await popover.getProperty("open")).not.toBe(true);
+    });
```

Comment:
This is cool and more inline with how Chrome's native time input works.  This will also setup users to expect the focus to be more on the input than in the popup, which will transition usage nicely over to the popover-less `input-time` component.

Reply:
Awesome! Thanks for the note.